src/main/java/com/minecolonies/api/tools/registry/IToolTypeRegistry.java

```diff
@@ -0,0 +1,12 @@
+package com.minecolonies.api.tools.registry;
+
+import com.minecolonies.api.IMinecoloniesAPI;
+import net.minecraftforge.registries.IForgeRegistry;
+
+public class IToolTypeRegistry
```

Comment:
You didn't strictly have to create this interface, because we're trying to actually consolidate logic, you can keep this method just fine within `ModToolTypes`, but regardless what you do, javadoc is required.

Reply:
Done.